QuickChat/Configuration/GeneralFrame.lua

```diff
@@ -0,0 +1,16 @@
+local frame = CreateFrame("Frame", "QuickChat_GenCfgFrame", QuickChat.mainConfigurationFrame)
+frame.name = "General"
+frame.parent = QuickChat.mainConfigurationFrame.name
+InterfaceOptions_AddCategory(frame)
+QuickChat.generalConfigurationFrame = frame
+
+function frame:OnAddonLoaded()
+    frame:CreateCheckButton("LockFramePosition", "Lock frame position", QC_Settings.lockFramePosition, function(checked)
+        QC_Settings.lockFramePosition = checked
+    end)
+end
+
+local previousButton = nil
```

Comment:
All this "previousButton" isn't used yet, but I think we'll add more stuff later.